docs/tutorials/spin_with_pwm_generator.rst

```diff
@@ -0,0 +1,94 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _spin_with_pwm_generator_tutorial:
+
+###############################################
+Module Control With a Generic PWM Generator
+###############################################
+
+This tutorial is designed to help guide you through spinning your module with a generic PWM generation device. In this example, we will use a `Pololu USB Servo Controller with their Maestro software <https://www.pololu.com/category/102/maestro-usb-servo-controllers>`_, 
+but the same logic applies to all PWM generators.
+
+*************************
+Hardware Configuration
+*************************
+
+All Vertiq modules accept PWM inputs on their RX/PWM In pins. You will need to connect your PWM generator's output to the module's RX pin, and tie the two devices' ground connections together. Please refer to your module's family page to find specific pinouts.
+
+.. image:: ../_static/tutorial_images/calibration_tutorial/basic_pwm_connection.png
+
+**********************
+Module Configuration
+**********************
+
+We are going to configure our module exactly as we would for integration with a flight controller with our module in :ref:`PWM mode <throttle_pwm_mode>`. To do this, first, ensure that 

```

Comment:
Maybe worth just pointing out one more time here that PWM mode is not the same as using Standard PWM for communication. Just so people know its not the only mode you can use with it, its just one of them. Just because everyone always gets confused by these names...